kem/src/lib.rs

```diff
@@ -9,11 +9,24 @@
 #![forbid(unsafe_code)]
 #![warn(missing_docs, unused_qualifications, missing_debug_implementations)]
 
-#[cfg(feature = "std")]
-extern crate std;
+/// A value that can be encapsulated to. Often, this will just be a public key. However, it can
+/// also be a bundle of public keys, or it can include a sender's private key for authenticated
+/// encapsulation.
+pub trait Encapsulate<EK, SS> {
+    /// Encapsulation error
+    type Error;
```

Comment:
It would be nice to bound this by `Debug` (`type Error: Debug;`) so that `Result::unwrap()` is happy.